build.gradle

```diff
@@ -55,7 +54,6 @@ minecraft {
             property 'forge.logging.console.level', 'debug'
             property 'mixin.env.remapRefMap', 'true'
             property 'mixin.env.refMapRemappingFile', "${projectDir}/build/createSrgToMcp/output.srg"
-            jvmArgs '-XX:+AllowEnhancedClassRedefinition'
```

Comment:
Perhaps it's a good idea to remove this because it does require using JBR. But it shouldn't be modified in this unrelated PR.

Reply:
The arg does exist. It requires using JBR for enhanced debugging. Basically, it allows you to hot reload classes and other things that the standard debugger does not allow. It means you don't have to restart the game when making changes outside of a method.

But it probably makes sense to remove for anyone who wants to contribute. It has been a source of headaches for some and I never documented this anywhere.